app/routes_customers.py

```diff
@@ -0,0 +1,92 @@
+from app.models.customer import Customer
+from app.models.rental import Rental
+from app.models.video import Video
+from flask import Blueprint, request, jsonify, make_response
+from app import db 
+
+customers_bp = Blueprint("customers", __name__, url_prefix="/customers")
+
+@customers_bp.route("", methods=["GET"], strict_slashes=False)
+def get_customers():
+    customers = Customer.query.all() 
+    customers_response = [] 
+
+    for customer in customers: 
+        customers_response.append(customer.to_json())
+
+    return jsonify(customers_response), 200
+
+@customers_bp.route("/<customer_id>", methods=["GET"], strict_slashes=False)
+def get_one_customer(customer_id):
+    customer = Customer.query.get(customer_id)
+    
+    if customer: 
+        return jsonify(customer.to_json()), 200
+    
+    return make_response("", 404)
```

Comment:
You can simplify this using `get_or_404`:

```suggestion
    customer = Customer.query.get_or_404(customer_id)
    
    return jsonify(customer.to_json()), 200
```